release.yml

```
@@ -0,0 +1,121 @@
+trigger: none
+pr: none
+
+parameters:
+  - name: test
+    type: boolean
+    default: true
```

nit: For consistency, keep list elements, `- {prop}`, aligned with the parent property. Thus,

```suggestion
parameters:
- name: test
  type: boolean
  default: true
```

You did this with `variables:` and `repositories:` and other stuff below. It still works if it is indented extra, but it just adds to confusion about YAML if they're all not the same.